accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -322,7 +326,7 @@ public static void main(String[] args)
     {
         int count = 1;
         int operations = 1000;
-        Long overrideSeed = -3309027396608571728L;
+        Long overrideSeed = -1320173654115300201L;
```

Comment:
please revert

Reply:
why?

Reply:
nope, `trunk` has this as `Long overrideSeed = -3309027396608571728L;`, so asking that we only include changes needed